lib/Slider/Slider.js

```diff
@@ -269,19 +269,14 @@ module.exports = kind(
 	* @private
 	*/
 	jumpWrapperComponents: [
-		{name: 'buttonLeft', kind: IconButton, backgroundOpacity: 'transparent', classes: 'moon-slider-button left', icon: 'arrowlargeleft', onSpotlightKeyDown: 'configureSpotlightHoldPulse', onSpotlightSelect: 'previous', ondown: 'downLeft', onholdpulse: 'holdLeft', ondragstart: 'preventDrag', defaultSpotlightDisappear: 'buttonRight'},
+		{name: 'buttonLeft', kind: IconButton, backgroundOpacity: 'transparent', classes: 'moon-slider-button left', icon: 'arrowlargeleft', onSpotlightKeyDown: 'showKnobStatus', onSpotlightKeyUp: 'hideKnobStatus', onSpotlightSelect: 'previous', ondown: 'downLeft', onup: 'hideKnobStatus', onholdpulse: 'holdLeft', ondragstart: 'preventDrag', defaultSpotlightDisappear: 'buttonRight'},
```

Comment:
Oh, I didn't catch the first feedback.
And, reducing event handlers is what I'd like to do.
I also moved {{showKnobStatus}} in {{next}} and {{previous}} but it could not fulfill new requirement.
You said, when okay key is released, popup should be closed.
But unfortunately, {{onSpotlightSelect}} event which call {{next}} and {{previous}} comes
after {{onSpotlightKeyUp}} event.
So we cannot match new requirement.

I'm still considering how can I match new requirement with reducing handlers